examples/gno.land/r/demo/groups/group.gno

```diff
@@ -0,0 +1,131 @@
+package groups
+
+import (
+	"gno.land/p/demo/avl"
+	"std"
+	"strconv"
+	"time"
+)
+
+type GroupID uint64
+type MemberID uint64
+
+func (gid GroupID) String() string {
+	return strconv.Itoa(int(gid))
+}
+
+func (mid MemberID) String() string {
+	return strconv.Itoa(int(mid))
+}
+
+type Member struct {
+	id        MemberID
+	address   string
```

Comment:
> maybe `address` can be of type `std.Address` instead of `string`

Reply:
Yes, it would also be consistent with `Group.creator`, which is an `std.Address` and not a `string`